spec/dependabot/shared_helpers_spec.rb

```diff
@@ -3,6 +3,22 @@
 require "dependabot/shared_helpers"
 
 RSpec.describe Dependabot::SharedHelpers do
+  describe ".in_a_temporary_directory" do
```

Comment:
Sneaked in a small tweak to the temp dir creation helper, happy to pull that out as part of another PR if you'd not like it to go in that direction.